package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@craco/craco": "^6.4.3",
```

Comment:
Should Craco be mentioned in PR notes? Even if it's just a Tailwind dependency?

Reply:
I think so - good catch!